CHANGELOG.rst

```diff
@@ -11,6 +11,11 @@ Added
   for password protected SSH key files. (improvement)
 
   Reported by Sibiraja L, Nick Maludy.
+* Append note to the output, if there are more entries for the following ``list`` CLIs: ``rule``,
+  ``execution``, ``rule-enforcment``, ``trace`` and ``trigger-instance`` as they are not displayed.
+  Default limit is 50. (improvement)
+
+  Reported by Eugen C. #3488
```

Comment:
Just `#3514 #3488` is enough there.

I think we include "reported by" if there was an external contribution.